src/content/tutorials/observability/alerting/configure-alertmanager/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+linkTitle: Configuring alerting
+title: How to configure alerting
+description: Guide explaining how to configure alerting in the Observability Platform.
+menu:
+  principal:
+    identifier: tutorials-observability-alerting-configuration
+    parent: tutorials-observability-alerting
+weight: 40
+last_review_date: 2025-04-24
+user_questions:
+  - How to configure Alertmanager?
+  - How to visualize the existing alerting configuration?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+The Giant Swarm Observability Platform provides an [alerting pipeline]({{< relref "/overview/observability/alerting/" >}}) that you can configure per tenant. This tutorial will guide you on this process.
+
+## Configure Alertmanager
+
+The Observability Platform uses Mimir Alertmanager as the central tool to provide its alerting capabilities.
+
+Once you have [configured your list of tenants]({{< relref "/tutorials/observability/multi-tenancy/" >}}, you can easily create an Alertmanager configuration for that tenant by creating the following secret on your management cluster.
```

Comment:
I don't think, that we ever use the terminology of "configuring a list of tenants" in the docs. It's more like "if you have listed a tenant in a Grafana Organisation". I think as it stands that might confuse people not having our insights. right? 
```suggestion
Once you have [listed a tenant in a Grafana Organisation]({{< relref "/tutorials/observability/multi-tenancy/" >}}, you can easily create an Alertmanager configuration for that tenant by creating the following secret on your management cluster.
```